### DISCUSSION TITLE
StreamDiffusion pipeline

### DISCUSSION THREAD

**Discussion by Jonathhhan** [leejet/stable-diffusion.cpp#201] [Ideas]
The StreamDiffusion pipeline seems to be very well optimized: https://github.com/cumulo-autumn/StreamDiffusion
I wonder, if it is possible to implement it in stable-diffusion.cpp.

**Comment by FSSRepo** [comment]
It requires knowing how to use TensorRT, which is a library that works only with Python. It would be best if one of those who contributed to that project helps in adding many optimizations in sd.cpp.

**Comment by Jonathhhan** [comment]
@FSSRepo there is also an official c++ API:
https://docs.nvidia.com/deeplearning/tensorrt/api/c_api/index.html
https://github.com/cyrusbehr/tensorrt-cpp-api
